source/views/sis/components/formatted-line.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import React from 'react'
+import {Text} from 'react-native'
+
+type Props = {
+	items: Array<string>,
+}
+
+export class FormattedLine extends React.PureComponent<Props> {
```

Comment:
If this only takes an array of strings, why not just use `myStringArray.join('•')`?

Reply:
lol because I'm stupid